pkg/chaosctl/cmd/physicalmachine.go

```diff
@@ -16,17 +16,12 @@
 package cmd
 
 import (
-	"fmt"
-	"os"
-
-	"github.com/go-logr/logr"
 	"github.com/spf13/cobra"
 
 	"github.com/chaos-mesh/chaos-mesh/pkg/chaosctl/physicalmachine"
 )
 
-func NewPhysicalMachineCommand(logger logr.Logger) (*cobra.Command, error) {
-
+func NewPhysicalMachineCommand() (*cobra.Command, error) {
```

Comment:
Because the logger is only used to print errors, which is unfriendly for a cli application. I prefer return errors and print out by the pretty printer.

Reply:
Refer to https://github.com/chaos-mesh/chaos-mesh/pull/2243#issuecomment-952522490